webapp/style/_vars.scss

```diff
@@ -23,6 +23,28 @@ $black: #2f3138;
 $yellowHighlight: #f8f6cc;
 $white: #ffffff;
 
+:root {
+  --red: #ed5b46;
+  --pink: #f7dfe3;
+  --green: #84ac67;
+  --greenLight: #f0f2e7;
+  --yellow: #fee997;
+
+  --grey: #e7e9ee;
+  --greyAppBg: #f6f7f9;
+  --greyBorder: #d1d1dd;
+  --blue: #3885ca;
+  --blueLight: #c5dbf0;
+  --blueLightFocus: #edf4fa;
+  --blueLightActive: #9dc3e5;
+  --blueDark: #265e8f;
+  --orange: #eba444;
+  --orangeLight: #f9ccb2;
+  --black: #2f3138;
+  --yellowHighlight: #f8f6cc;
+  --white: #ffffff;
+}
```

Comment:
this doesn't go here. to main style, also use the vars instead of explicit colors

Reply:
Using the Sass var the value appears null 
![Captura de pantalla 2020-11-23 a las 10 29 13](https://user-images.githubusercontent.com/11272068/99946490-bde19300-2d76-11eb-8d2e-20a1ce892430.png)
<img width="292" alt="Captura de pantalla 2020-11-23 a las 10 29 09" src="https://user-images.githubusercontent.com/11272068/99946535-cd60dc00-2d76-11eb-8600-954079da7d2e.png">

Reply:
fix it then.
you could eventually add these to a new sass file (cssVars)

Reply:
Solved!